ath.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+cd $(dirname $0)
+
+# https://github.com/jenkinsci/acceptance-test-harness/releases
+ATH_VERSION=5458.v911b_2f0818ee
```

Comment:
```suggestion
export ATH_VERSION=5458.v911b_2f0818ee
```

to use the "simplified" Docker command